x/gov/keeper/proposal.go

```diff
@@ -51,18 +51,29 @@ func (keeper Keeper) SubmitProposal(ctx sdk.Context, content types.Content) (typ
 // GetProposal get proposal from store by ProposalID
 func (keeper Keeper) GetProposal(ctx sdk.Context, proposalID uint64) (proposal types.Proposal, ok bool) {
```

Comment:
Looks like there is no need for named returns.

Reply:
Correct, we can remove the named returns now (which I generally advise against).